spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/StructAccessor.java

```diff
@@ -79,28 +79,28 @@
 		this.columnNamesIndex = indexColumnNames();
 	}
 
-	public Object getSingleValue(String colName) {
+	Object getSingleValue(String colName) {
```

Comment:
I see you're removing the visibility modifier in a bunch of these methods.
Make sure they're not in any public-facing API, where it would make sense they're not being called from within the project.